tsconfig.json

```diff
@@ -1,5 +1,7 @@
 {
   "compilerOptions": {
+    "experimentalDecorators": true,
+    "experimentalDecorators": true,
```

Comment:
Why is this line duplicated?

Reply:
Good catch, it shouldn't be. I'll fix that!